### PR TITLE
Create depthk.py

### PR DESCRIPTION
**Pull request by WilliameRocha**

### CODE REVIEW

File: benchexec/tools/depthk.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python
```
Comment (PhilippWendler) [inline benchexec/tools/depthk.py]: This script is not executable, so this line is superfluous.

File: benchexec/tools/depthk.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+            status = result.RESULT_FALSE_MEMTRACK
+        elif 'FALSE' in output:
+            status = result.RESULT_FALSE_REACH
+        else:
+            status = result.RESULT_UNKNOWN
+
+        # else:
+        #    status = result.RESULT_ERROR
+
+        return status
```
Comment (PhilippWendler) [inline benchexec/tools/depthk.py]: This method should not return an empty string or `None`. Please change that `result.RESULT_ERROR` is returned if no result can be determined (as it was before).

File: benchexec/tools/depthk.py [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+            if identifier == 'k' and line.startswith('Bound k:'):
+                matchbound = re.search(r'Bound k:(.*)', line)
+                return matchbound.group(1).strip()
+            if identifier == 'Step' and line.startswith('Solution by:'):
+                matchstep = re.search(r'Solution by:(.*)', line)
+                return matchstep.group(1).strip()
+
+        return '-'
+
+    def addColumnValues(self, output, columns):
```
Comment (PhilippWendler) [inline benchexec/tools/depthk.py]: The method `addColumnValues` was only used a long time ago. It was replaced by `get_value_from_output`, so please remove it.

**Comment by dbeyer** [comment]
@WilliameRocha What is the ETA for the fixes?

**Comment by WilliameRocha** [comment]
Ok, I will create a new file and submit it.

Thanks

**Pull request closed without merging**